src/slist.c

```diff
@@ -1 +1,324 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <stdbool.h>
+
+#include "macro_utils/macro_utils.h"
+
+#include "c_logging/xlogging.h"
+
 #include "c_util/slist.h"
+
+bool slist_initialize(PSINGLYLINKEDLIST_ENTRY list_head)
+{
+	bool result;
```

Comment:
>	 [](http://example.com/codeflow?start=0&length=1)

Can we change to using spaces please?
You probably need to configure your VS/VSCode to use spaces for indentation instead of Tabs.

Reply:
done